CHANGELOG.md

```diff
@@ -11,8 +20,8 @@
   * [FIX] Customizations in the user-values-prom-operator.yaml file under the alertmanagerSpec will now be applied correctly.
 
 * **Logging**
-  *  [UPGRADE] Opensearch and Opensearch Dashboards has been upgraded from 1.32 to 1.35
-  *  [UPGRADE] Elasticsearch Exporter has been upgraded from 1.2.1 to 1.5.0
+  * [UPGRADE] Opensearch and Opensearch Dashboards has been upgraded from 1.32 to 1.35
+  * [UPGRADE] Elasticsearch Exporter has been upgraded from 1.2.1 to 1.5.0
 
```

Comment:
Out of curiosity: was this a human-made change or did the editor do this as part of an automatic cleanup/linter process?

Reply:
This was actually a manual change, just happened to notice it and made the adjustment